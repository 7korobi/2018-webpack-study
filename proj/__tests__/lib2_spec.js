import lib from '../src/js/lib2';

describe("lib2", ()=> {
  test('snapshot', ()=> {
    expect(
      lib(process)
    ).toMatchSnapshot()
  })
})

