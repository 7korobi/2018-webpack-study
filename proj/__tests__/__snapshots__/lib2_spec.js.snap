// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib2 snapshot 1`] = `
Object {
  "env": Object {
    "ANDROID_HOME": "/Users/7korobi/Library/Android/sdk",
    "Apple_PubSub_Socket_Render": "/private/tmp/com.apple.launchd.ME6VTX7C0o/Render",
    "COMMAND_MODE": "unix2003",
    "HOME": "/Users/7korobi",
    "INIT_CWD": "/Users/7korobi/Dropbox/Public/2018-webpack-study/proj",
    "JEST_WORKER_ID": "1",
    "LANG": "ja_JP.UTF-8",
    "LOGNAME": "7korobi",
    "NODE": "/Users/7korobi/.nodebrew/node/v10.11.0/bin/node",
    "NODE_ENV": "test",
    "PATH": "/Users/7korobi/Dropbox/Public/2018-webpack-study/proj/node_modules/.bin:/Users/7korobi/.config/yarn/link/node_modules/.bin:/Users/7korobi/Dropbox/Public/2018-webpack-study/proj/node_modules/.bin:/Users/7korobi/.config/yarn/link/node_modules/.bin:/Users/7korobi/.nodebrew/node/v10.11.0/libexec/lib/node_modules/npm/bin/node-gyp-bin:/Users/7korobi/.nodebrew/node/v10.11.0/lib/node_modules/npm/bin/node-gyp-bin:/Users/7korobi/.nodebrew/node/v10.11.0/bin/node_modules/npm/bin/node-gyp-bin:/Users/7korobi/.nodebrew/node/v10.11.0/bin:/Users/7korobi/.rbenv/shims:/Users/7korobi/.nodebrew/current/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/7korobi/.rbenv/shims:/Users/7korobi/.nodebrew/current/bin:/Users/7korobi/Library/Android/sdk/tools:/Users/7korobi/Library/Android/sdk/tools/bin:/Users/7korobi/Library/Android/sdk/platform-tools:/Users/7korobi/Library/Android/sdk/emulator:/Users/7korobi/Library/Android/sdk/build-tools/28.0.3:/Users/7korobi/Library/Android/sdk/tools:/Users/7korobi/Library/Android/sdk/tools/bin:/Users/7korobi/Library/Android/sdk/platform-tools:/Users/7korobi/Library/Android/sdk/emulator:/Users/7korobi/Library/Android/sdk/build-tools/28.0.3",
    "PWD": "/Users/7korobi/Dropbox/Public/2018-webpack-study/proj",
    "RBENV_SHELL": "bash",
    "SECURITYSESSIONID": "186a7",
    "SHELL": "/bin/bash",
    "SHLVL": "3",
    "SSH_AUTH_SOCK": "/private/tmp/com.apple.launchd.Y7ISbiJ2TB/Listeners",
    "TERM": "xterm-256color",
    "TERM_PROGRAM": "vscode",
    "TERM_PROGRAM_VERSION": "1.29.1",
    "TMPDIR": "/var/folders/vx/c90hzdjs4mbc54bykgkq0h900000gp/T/",
    "USER": "7korobi",
    "XPC_FLAGS": "0x0",
    "XPC_SERVICE_NAME": "0",
    "YARN_WRAP_OUTPUT": "false",
    "_": "/Users/7korobi/Dropbox/Public/2018-webpack-study/proj/node_modules/.bin/jest",
    "__CF_USER_TEXT_ENCODING": "0x1F6:0x1:0xE",
    "npm_config_argv": "{\\"remain\\":[],\\"cooked\\":[\\"run\\",\\"jest\\"],\\"original\\":[\\"jest\\"]}",
    "npm_config_bin_links": "true",
    "npm_config_ignore_optional": "",
    "npm_config_ignore_scripts": "",
    "npm_config_init_license": "MIT",
    "npm_config_init_version": "1.0.0",
    "npm_config_registry": "https://registry.yarnpkg.com",
    "npm_config_save_prefix": "^",
    "npm_config_strict_ssl": "true",
    "npm_config_user_agent": "yarn/1.10.1 npm/? node/v10.11.0 darwin x64",
    "npm_config_version_commit_hooks": "true",
    "npm_config_version_git_message": "v%s",
    "npm_config_version_git_sign": "",
    "npm_config_version_git_tag": "true",
    "npm_config_version_tag_prefix": "v",
    "npm_execpath": "/Users/7korobi/.nodebrew/node/v10.11.0/lib/node_modules/yarn/bin/yarn.js",
    "npm_lifecycle_event": "jest",
    "npm_node_execpath": "/Users/7korobi/.nodebrew/node/v10.11.0/bin/node",
    "npm_package_author_email": "7korobi@gmail.com",
    "npm_package_author_name": "7korobi",
    "npm_package_dependencies_mini_css_extract_plugin": "^0.4.5",
    "npm_package_devDependencies__webpack_cli_init": "^0.1.0",
    "npm_package_devDependencies_autoprefixer": "^9.4.2",
    "npm_package_devDependencies_babel_core": "^6.26.3",
    "npm_package_devDependencies_babel_jest": "^23.4.2",
    "npm_package_devDependencies_babel_loader": "^8.0.4",
    "npm_package_devDependencies_babel_preset_env": "^1.7.0",
    "npm_package_devDependencies_babel_preset_es2015": "^6.24.1",
    "npm_package_devDependencies_babel_preset_react": "^6.24.1",
    "npm_package_devDependencies_css_loader": "^1.0.1",
    "npm_package_devDependencies_extract_text_webpack_plugin": "^4.0.0-beta.0",
    "npm_package_devDependencies_html_webpack_plugin": "^3.2.0",
    "npm_package_devDependencies_jest": "^23.6.0",
    "npm_package_devDependencies_node_sass": "^4.10.0",
    "npm_package_devDependencies_postcss": "^7.0.6",
    "npm_package_devDependencies_postcss_loader": "^3.0.0",
    "npm_package_devDependencies_sass_loader": "^7.1.0",
    "npm_package_devDependencies_style_loader": "^0.21.0",
    "npm_package_devDependencies_stylelint": "^9.3.0",
    "npm_package_devDependencies_stylelint_config_standard": "^18.2.0",
    "npm_package_devDependencies_stylelint_webpack_plugin": "^0.10.5",
    "npm_package_devDependencies_webpack": "^4.27.0",
    "npm_package_devDependencies_webpack_cli": "^3.1.2",
    "npm_package_devDependencies_webpack_serve": "^2.0.2",
    "npm_package_license": "MIT",
    "npm_package_main": "index.js",
    "npm_package_name": "demo-webpack",
    "npm_package_scripts_app1": "webpack --config webpack.2-1.js --mode production",
    "npm_package_scripts_app2": "webpack --config webpack.2-2.js --mode production",
    "npm_package_scripts_app3": "webpack --config webpack.3-1.js --mode production",
    "npm_package_scripts_app4": "webpack --config webpack.3-2.js --mode production",
    "npm_package_scripts_autoprefixer_info": "autoprefixer --info",
    "npm_package_scripts_dev": "webpack-serve --config webpack.2-2.js",
    "npm_package_version": "1.0.0",
  },
  "versions": Object {
    "ares": "1.14.0",
    "cldr": "33.1",
    "http_parser": "2.8.0",
    "icu": "62.1",
    "modules": "64",
    "napi": "3",
    "nghttp2": "1.33.0",
    "node": "10.11.0",
    "openssl": "1.1.0i",
    "tz": "2018e",
    "unicode": "11.0",
    "uv": "1.23.0",
    "v8": "6.8.275.32-node.28",
    "zlib": "1.2.11",
  },
}
`;
